{"uuid": "65072bb5-c1b1-4f59-b463-59399ba7be19", "children": ["d35a6bdb-9ba6-441a-a2b1-f9d704c19f03", "9c9204b1-edf6-4740-b92d-59349dcca9df"], "befores": [{"name": "init_desktop_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Failed to locate opened application window with appId: Microsoft.WindowsCalculator_8wekyb3d8bbwe!App, and processId: 11256\n\n", "trace": "  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Automation\\Final_Proj_Aut_Py\\test_cases\\conftest.py\", line 107, in init_desktop_driver\n    edriver = get_desktop_driver()\n              ^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Automation\\Final_Proj_Aut_Py\\test_cases\\conftest.py\", line 168, in get_desktop_driver\n    driver = appium.webdriver.Remote(get_data('WinAppDriverService'), dc)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\appium\\webdriver\\webdriver.py\", line 267, in __init__\n    super().__init__(\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\appium\\webdriver\\webdriver.py\", line 357, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1684578135899, "stop": 1684578141652}], "start": 1684578135899, "stop": 1684578142264}